yrs/src/block.rs

```diff
@@ -115,6 +131,7 @@ impl BlockPtr {
                         right_origin: item.right_origin.clone(),
                         content,
                         parent: item.parent.clone(),
+                        moved: item.moved.clone(),
```

Comment:
Moved field is now present on every item, adding 4-8B extra memory footprint per block.